dsl/git_push_test.jmx

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
```

Comment:
Oh, I see. Hadn't got as far as running `jmeter` yet. If I'm not expected to spelunk in XML then I think we can omit this file.